src/theme.js

```diff
@@ -104,6 +104,17 @@ const theme = {
       background: 'rgba(225, 225, 225, 0.75)'
     }
   },
+  radioButton: {
+    extend: props => css`
```

Comment:
[super minor] does this need to be a `props => {}` function? The CSS values seem pretty static. But hey, it works, and the `css()` call is there to prevent ye olde memory leakes, so all good. 👌